packages/cdk/lib/stacks/engines/nextflow-engine-stack.ts

```diff
@@ -1,31 +1,34 @@
-import { NestedStackProps } from "monocdk";
-import { Construct } from "constructs";
+import { Aws, Construct } from "monocdk";
 import { NextflowEngine } from "../../constructs/engines/nextflow/nextflow-engine";
 import { renderServiceWithContainer } from "../../util";
 import { EngineOptions } from "../../types";
 import { Bucket } from "monocdk/aws-s3";
 import { ApiProxy } from "../../constructs";
 import { LogGroup } from "monocdk/aws-logs";
-import { EngineOutputs, NestedEngineStack } from "./nested-engine-stack";
+import { EngineOutputs, EngineConstruct } from "./engine-construct";
 import { ILogGroup } from "monocdk/lib/aws-logs/lib/log-group";
 import { IJobQueue } from "monocdk/aws-batch";
 import { NextflowEngineRole } from "../../roles/nextflow-engine-role";
 import { NextflowAdapterRole } from "../../roles/nextflow-adapter-role";
 
-export interface NextflowEngineStackProps extends EngineOptions, NestedStackProps {
+export interface NextflowEngineStackProps extends EngineOptions {
+  /**
+   * AWS Batch JobQueue to use for running workflows.
+   */
+  readonly jobQueue: IJobQueue;
```

Comment:
I don't see this being used anywhere. Is it needed?

Reply:
Yup, its used a few times. Here is one: https://github.com/aws/amazon-genomics-cli/pull/150/files#diff-8623b226bf6f1a3639f3853bdccb79a30a41db59747e010dbe8fc601353f9273R38